hbw/app/assets/javascripts/hbw/components/form/string.js.jsx.coffee

```diff
@@ -85,28 +77,39 @@ modulejs.define 'HBWFormString', ['React', 'jQuery', 'HBWCallbacksMixin', 'HBWDe
           @trigger('hbw:form-submitting-failed')
 
     onLoadValidation: ->
-      if @validationRequired() && !!this.state.value
+      if @validationRequired() && !!@state.value
         @controlValidationTooltip(@isValid())
         @setValidationState()
 
     onChange: (event) ->
-      $el = jQuery(event.target)
-      @updateValue($el)
+      @updateValue(jQuery(event.target)[0], false)
```

Comment:
It seems we don't need to pass jQuery object to `updateValue`. `jQuery(event.target)[0]` --> `event.target`